src/posix/unix/i_specialpaths.cpp

```diff
@@ -203,7 +228,8 @@ FString M_GetConfigPath(bool for_reading)
 
 FString M_GetScreenshotsPath()
 {
-	return NicePath("$HOME/" GAME_DIR "/screenshots/");
+	FString path = GetEnvironmentVariable("XDG_CONFIG_HOME", FString("$HOME/")+GAME_DIR);
+	return path + "screenshots/";
```

Comment:
This needs a leading slash, in cae the env variable doesn't contain a trailing slash